app/routes/create.js

```diff
@@ -14,8 +14,9 @@ export default Route.extend(AuthenticatedRouteMixin, EventWizardMixin, {
         copyright           : this.store.createRecord('event-copyright'),
         stripeAuthorization : this.store.createRecord('stripe-authorization')
       }),
-      module : await this.get('store').queryRecord('module', {}),
-      types  : await this.store.query('event-type', {
+      module         : await this.get('store').queryRecord('module', {}),
+      verifiedStatus : this.get('authManager.currentUser').isVerified,
```

Comment:
Use ES6 getters. `this.authManager.currentUser.isVerified`

Reply:
okay